packages/angular_devkit/build_angular/src/webpack/plugins/javascript-optimizer-plugin.ts

```diff
@@ -85,19 +86,42 @@ export class JavaScriptOptimizerPlugin {
         },
         async (compilationAssets) => {
           const scriptsToOptimize = [];
+          const cache = compilation.getCache('JavaScriptOptimizerPlugin');
```

Comment:
```suggestion
          const cache = !!compilation.options.cache && compilation.getCache('JavaScriptOptimizerPlugin');
```
nit: checking here should avoid the `getCache` call if not needed.

Reply:
I was going for that but I did complicate the type in Line 100.

Will try to give it another shoot.